R/utils.R

```diff
@@ -164,11 +164,12 @@ is_first <- function(x) {
   seq_along(x) == 1L
 }
 
-handle_unmatched <- function(data, level_cols) {
+polish_output <- function(data, level_cols) {
   na_cols <- setdiff(level_cols, "companies_id")
   data |>
     prune_unmatched("risk_category", .by = "companies_id") |>
-    spread_na_across(na_cols, from = "risk_category")
+    spread_na_across(na_cols, from = "risk_category") |>
+    distinct()
```

Comment:
This is it. Because all indicators call `polish_output()` this spreads everywhere.